CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ Bugfixes:
   Patch by [Daraan](https://github.com/Daraan).
 - Fix `TypeError` when using `evaluate_forward_ref` on Python 3.10.1-2 and 3.9.8-10.
   Patch by [Daraan](https://github.com/Daraan).
+- Fix `TypeError` when taking the union of `typing_extensions.TypeAliasType` and a
```

Comment:
Too late for this section, I already released 4.13.1. You'll have to add a new "Unreleased" section. We'll either have to make another bugfix release or it'll have to wait for 4.14.0.

Reply:
Oeps